tests/test_decryption_key_index.rs

```diff
@@ -0,0 +1,60 @@
+use openmls::prelude::*;
+
+mod utils;
+
+use utils::managed_utils::*;
+
+#[test]
+fn test_decryption_key_index_computation() {
+    // Some basic setup functions for the managed group.
+    let handshake_message_format = HandshakeMessageFormat::Plaintext;
+    let update_policy = UpdatePolicy::default();
+    let callbacks = default_callbacks::default_callbacks();
+    let managed_group_config =
+        ManagedGroupConfig::new(handshake_message_format, update_policy, callbacks);
+    let number_of_clients = 20;
+    let setup = ManagedTestSetup::new(managed_group_config, number_of_clients);
+    setup.create_clients();
+
+    for ciphersuite in Config::supported_ciphersuites() {
```

Comment:
Please make one test for each cipher suite.

Reply:
Is there a particular reason why this would make sense in this test in particular (or in general)? For example, in  `test_group`, we also have a single test function that simply iterates over each ciphersuite in the same way. Doesn't the loop make sense for tests that we don't expect to be particularly ciphersuite-dependent?

Reply:
Having them in one function makes debugging way harder (you have no idea which ciphersuite breaks). I want to have all tests parameterized on the cipher suite eventually. It just needs some boilerplate macros.

Reply:
Macros would make sense. But until then, can't we just stick with loops in places where it's not ciphersuite specific? We have to touch the code anyway when we have the macro.

For example, in follow-up PR #312 , I'm going to add the tests for the interop scenarios. Doing one test per ciphersuite would be quite a bit more code.